storybook/qmlTests/tests/tst_OnboardingLayout.qml

```diff
@@ -976,31 +981,33 @@ Item {
 
                 const passwordInput = findChild(page, "loginPasswordInput")
                 verify(!!passwordInput)
-                tryCompare(passwordInput, "activeFocus", true)
+                tryCompare(passwordInput, "activeFocus", !data.biometrics)
```

Comment:
Hmm, something's fishy here :) The biometrics popup should not steal focus imo; the keyboard/input focus should always stay where the password/pin input is

Reply:
I changed the behavior in a way that when the biometrics popup is invoked, the login screen is disabled, there is no possibility to enter password manually or change account via selector.

Imo it's cleaner and easier. Otherwise user can login on it's own by entering password manually with the biometrics popup still open. Or the account may be changed in the meantime.

Reply:
That's perfectly valid scenario: 

![image](https://github.com/user-attachments/assets/e0027597-dc80-4560-8ed5-b3a4901ed21f)

Even when the bio popup is visible, the user can still enter the password manually ^^

Reply:
Ok, as agreed, biometrics should not be blocking, fixed.